Design-Docs/glossary.md

```diff
@@ -0,0 +1,143 @@
+---
+title: SCS Definitions (Draft)
+version: 2021-09-09-001
+authors: Matthias Hamm
+state: v1.0 (for R0)
+---
+
+Glossary - Naming conventions for SCS (Draft) 
+
+# Motivation
+
+In order to have a consistent naming scheme in SCS environments, we need a definition of the different terms. 
+
+# Proposal
+
+This is a draft version, which is free to be discussed
+
+## Definition of a Region
+
+An OpenStack/SCS Region consists of at least one or more Availability Zones that share a Control Plane with their services. As a result, they share an API, as well as a CEPH cluster, i.e. a data store. Within the region, any Layer 2 networks are available to the user.
+
+Regions can be federated when the SCS code is ready.
+
+A region is located in one fire compartment.
+
+## Definition of a Avialbility Zone
+
+An Availability Zone is a (physical) group of multiple compute nodes, controlled by the region's control plane that provides the API and interface.
+
+An Availability Zone allows OpenStack compute hosts to be divided into logical groups and provides a form of physical isolation and redundancy from other Availability Zones, for example by using a separate power supply or network devices.
+
+When users provision resources, they can specify in which Availability Zone their instance should be created. In this way, cloud customers can ensure that their application resources are distributed across different computers to achieve high availability in the event of a hardware failure.
+
```

Comment:
This doesn't map to my definition of AZ for openstack. You are focusing on compute, while AZ is a notion that nova, cinder, and neutron have.
Next to that, that definition of AZ brings questions regarding glance.